src/main/java/com/adamk33n3r/runelite/watchdog/ui/ComparableNumber.java

```diff
@@ -21,7 +21,7 @@ public ComparableNumber(
         this.number = PanelUtils.createSpinner(val, min, max, step, onNumChange);
 
         this.setLayout(new DynamicGridLayout(1, 2, 5, 5));
-        this.comparator.setPreferredSize(new Dimension(50, 0));
+        this.comparator.setPreferredSize(new Dimension(100, 0));
```

Comment:
I actually have a better fix for this issue in #143 that I forgot to merge.

Reply:
Oh fantastic! I'll remove.